airflow/gcp/operators/tasks.py

```diff
@@ -603,21 +609,21 @@ class CloudTasksTaskCreateOperator(BaseOperator):
     )
 
     @apply_defaults
-    def __init__(
+    def __init__(  # pylint: disable=too-many-arguments
         self,
-        location,
-        queue_name,
-        task,
-        project_id=None,
-        task_name=None,
-        response_view=None,
-        retry=None,
-        timeout=None,
-        metadata=None,
-        gcp_conn_id="google_cloud_default",
+        location: str,
+        queue_name: str,
+        task: Union[Dict, Task],
+        project_id: Optional[str] = None,
+        task_name: Optional[str] = None,
+        response_view: Task.View = None,
```

Comment:
Can't we use some generic type here?

Reply:
Actually @mik-laj pointed out there is mistake in the Google documentaion, it should be `enums.Task.View` and it seems valid. @mik-laj could you confirm?

Reply:
And it should be `Optional` what I've added as well.